include/seqan3/core/debug_stream/debug_stream_type.hpp

```diff
@@ -151,6 +169,15 @@ class debug_stream_type
             *stream << v;
         return *this;
     }
+#else
+
+    template<typename T>
+    friend struct debug_stream_printer;
+
+    template<typename T>
+    friend struct std_printer;
```

Comment:
This is only needed since we have no API to access the underlying stream of a `debug_stream`